docs/topics/items.rst

```diff
@@ -183,6 +174,33 @@ Creating items from dicts::
         ...
     KeyError: 'Product does not support field: lala'
 
+Copying items::
+
+    >>> product['tags'] = ['tag1']
+
+    >>> product2 = Product(product)
+    >>> print(product2)
+    Product(name='Desktop PC', price=1000, tags=['tag1'])
+
+    >>> product3 = product.copy()
+    >>> print(product3)
+    Product(name='Desktop PC', price=1000, tags=['tag1'])
+
+    >>> product4 = product.deepcopy()
```

Comment:
Hey, what do you think about providing some text describing it, not only an example? It'd be nice to mention caveats of using `Product(product)` or `product.copy()`, and tell where user should prefer which way of copying.